pytripgui/canvas_vc/plotter/images/patient_image_base.py

```diff
@@ -0,0 +1,26 @@
+from abc import ABC, abstractmethod
+
+from matplotlib.image import AxesImage
+
+
+class PatientImageBase(ABC):

```

Comment:
Add a comment describing responsibility of this class

Reply:
done